.gitignore

```diff
@@ -56,9 +56,9 @@ typings/
 
 # dotenv environment variables file
 .env
-
+config.env
 # next.js build output
 .next
 .node_modules
 /client/build
-config.env
\ No newline at end of file
+config.env
```

Comment:
No need to ignore `config.env` two times in the same file

Reply:
delete one statement from .gitignore